Monitron.IM_RPC/Command.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Monitron.ImRpc
+{
+    public class Command
+    {
+        public string Name { get; set; }
+        public List<string> Args { get; set; }
```

Comment:
Use IList or ICollection for public properties to allow the implementation to change in the future.